app/src/main/res/layout/fragment_profile_new.xml

```diff
@@ -218,7 +217,7 @@
                         fontPath="fonts/Roboto-Light.ttf"
                         android:layout_width="match_parent"
                         android:layout_height="wrap_content"
-                        android:lineSpacingExtra="3.3sp"
+                        android:lineSpacingMultiplier="1.6"
```

Comment:
why have we changed it?

Reply:
to match with WebView line spacing which is `1.6 em`